docs/source/WritingTests.rst

```diff
@@ -1729,6 +1729,9 @@ tests:
 | `***`                       | All variables from --mux-yaml         | TIMEOUT=60; IO_WORKERS=10; VM_BYTES=512M; ...                                                       |
 +-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
 
+.. warning:: ``AVOCADO_TEST_SRCDIR`` is deprecated and will be removed
+             soon.  Please use ``AVOCADO_TEST_WORKDIR`` instead.
```

Comment:
nice, did you consider adding `*` into the matrix? (or footnote)

Reply:
I'm moving that line to the be the last line, and by doing so, being close to the "warning" lines.